project_admin/templates/project_admin/home.html

```diff
@@ -1,6 +1,15 @@
 {% extends 'main/application.html' %}
 
 {% block content %}
+{% if messages %}
+
+    {% for message in messages %}
```

Comment:
That's a very nice solution! Do you think it would make sense to break this out into the `main/application.html` template so that there's no need to specify this in all templates individually?

Reply:
Oh yes, didn't think of that flow. It definitely is less abrupt and more flexible than the current approach. Also the reason why I put it in project_admin/home.html was because the approach that I'm currently using doesn't let you explore the application right. You can't check the app unless you've set the mandatory fields. But yes, if we change approaches, it'll make more sense to put it in application.html.